xmipptomo/protocols/protocol_phantom_subtomo.py

```diff
@@ -49,28 +49,39 @@ def _defineParams(self, form):
         form.addParam('inputVolume', PointerParam, pointerClass="Volume", label='Input volume',
-                      condition="option==0", help="Volume used as 'base' phantom")
+                      condition="option==0", help="Volume used as 'base' phantom", allowsNull=True)
         form.addParam('create', TextParam, label='Create phantom', condition="option==1",
                       default='40 40 40 0\ncyl + 1 0 0 0 15 15 2 0 0 0\nsph + 1 0 0 5 2\ncyl + 1 0 0 -5 2 2 10 0 90 0\n'
                               'sph + 1 0 -5 5 2',
                       help="create a phantom description: x y z backgroundValue geometry(cyl, sph...) +(superimpose) "
-                           "desnsityValue origin radius height rot tilt psi")
+                           "density value origin radius height rot tilt psi. More info at https://web.archive.org/web/20180813105422/http://xmipp.cnb.csic.es/twiki/bin/view/Xmipp/FileFormats#Phantom_metadata_file")
         form.addParam('sampling', FloatParam, label='Sampling rate', default=4)
+        form.addParam('nsubtomos', IntParam, label='Number of subtomograms', default=50,
+                      help="How many phantom subtomograms")
         form.addParam('mwfilter', BooleanParam, label='Apply missing wedge?', default=False,
                       help='Apply a filter to simulate the missing wedge along Y axis.')
         form.addParam('mwangle', IntParam, label='Missing wedge angle', default=60, condition='mwfilter==True',
                       help='Missing wedge (along y) for data between +- this angle.')
-
-        form.addSection(label='Transform')
-        form.addParam('nsubtomos', IntParam, label='Number of subtomograms', default=50,
-                      help="How many phantom subtomograms")
+        # Angles
+        form.addSection(label='Rotation')
         form.addParam('rotmin', IntParam, label='Min rot angle', default=0,
                       help='Minimum and maximum range for each Euler angle in degrees')
         form.addParam('rotmax', IntParam, label='Max rot angle', default=60)
         form.addParam('tiltmin', IntParam, label='Min tilt angle', default=0)
         form.addParam('tiltmax', IntParam, label='Max tilt angle', default=60)
         form.addParam('psimin', IntParam, label='Min psi angle', default=0)
         form.addParam('psimax', IntParam, label='Max psi angle', default=60)
+
+        # Shifts
+        form.addSection(label='Shifts')
+        form.addParam('xmin', IntParam, label='Min shift in x', default=0)
+        form.addParam('xmax', IntParam, label='Max shift in x', default=5)
+        form.addParam('ymin', IntParam, label='Min shift in y', default=0)
+        form.addParam('ymax', IntParam, label='Max shift in y', default=5)
+        form.addParam('zmin', IntParam, label='Min shift in z', default=0)
+        form.addParam('zmax', IntParam, label='Max shift in z', default=5)
+
+        form.addSection(label='Others')
```

Comment:
Maybe we can call this section "coordinates" or just "coords". This is just a suggestion, as you prefer.